CONTRIBUTING.md

```diff
@@ -2,44 +2,48 @@
 
```

Comment:
Please move the documentation tidy up to a separate pull request, I'd like this one to stay focussed on reno.